test/e2e/README.md

```diff
@@ -104,6 +165,13 @@ For images deployed in GCR, a docker tag is mandatory to avoid issues with using
 ./test/upload-test-images.sh ./vendor/knative.dev/eventing/test/test_images/ e2e
 ```
 
+To run the script for all end to end test images:
+
+```bash
+./test/upload-test-images.sh ./test/test_images
+./test/upload-test-images.sh ./vendor/knative.dev/eventing/test/test_images/
```

Comment:
https://github.com/google/knative-gcp/blob/443a9e865c8e0e1169aaa5da766a0898c555dfa2/test/e2e-common.sh#L63 needs to be run before upload eventing test images

Reply:
Does this work in Mac? When I tried to run it, it failed with 
`sed: 1: "vendor/knative.dev/even ...": invalid command code v`